pip/iminuit.file

```diff
@@ -0,0 +1 @@
+Requires: py3-numpy py3-matplotlib py3-scipy py3-ipywidgets
```

Comment:
@DickyChant , do `iminuit` actually need `py3-scipy py3-ipywidgets` dependencies? I do not see any ref to these package in `imniut`

```
> pwd
.../py3-iminuit/1.2-9722ab16622cfb12a6caf4da6d1edccd
> grep -R -i -E 'scipy|ipywidgets'  .
./lib/python3.9/site-packages/iminuit/__init__.py:# http://docs.scipy.org/doc/numpy/reference/generated/numpy.testing.Tester.test.html
```

Reply:
nice catch I didn't investigate clearly but just follow https://scikit-hep.org/iminuit/ and adds as much optional dependencies as possible since I thought in our real world case have these in well defined sequence could make it more robust...